src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1699,7 +1699,7 @@ because xbuild doesn't support framework reference assemblies.
 	/>
 
 	<Aapt2Link
-		Condition="'$(_AndroidResourceDesignerFile)' != '' And '$(_AndroidUseAapt2)' == 'True' And Exists ('$(_AndroidLibraryFlatArchivesDirectory)\compiled.flata')"
+		Condition=" '$(_AndroidResourceDesignerFile)' != '' And '$(_AndroidUseAapt2)' == 'True' "
```

Comment:
Do we have a test (or have we tested) what happens with the Aapt2Link call where there are NO .flata files are all in the flata directory or if the flata directory does not exist ?

Reply:
I tried making a test with no resources, and it seems like we at least will always have 1 `flata` file. It seems an app icon is required:
```
obj\Debug\android\manifest\AndroidManifest.xml(8,0): error APT0000: resource drawable/icon (aka UnnamedProject.UnnamedProject:drawable/icon) not found.
```
It looks like we always put this in the `obj\Debug\android\AndroidManifest.xml`:
```xml
<?xml version="1.0" encoding="utf-8"?>
<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="UnnamedProject.UnnamedProject">
  <!--suppress UsesMinSdkAttributes-->
  <uses-sdk android:minSdkVersion="16" />
  <uses-permission android:name="android.permission.INTERNET" />
  <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
  <application android:label="UnnamedProject" android:name="android.app.Application" android:allowBackup="true" android:icon="@drawable/icon" android:debuggable="true">
...
```
The manifest I'm using just has:
```xml
<?xml version="1.0" encoding="utf-8"?>
<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="UnnamedProject.UnnamedProject">
	<uses-sdk />
	<application android:label="UnnamedProject">
	</application>
</manifest>
```

Reply:
ok thats cool. So we don't need to worry about that then.